Modify S4423: Learn-As-You-Code Migration (#2097)
SonarSource/rspec: rules/S4423/summary.adoc

> This vulnerability exposes encrypted data to an unnecessarily large number of

Reviewer: As mentioned in the other PR, I think "unnecessarily large" is not very precise, as it would be already enough if it exposes the data to one attack.

> This vulnerability exposes encrypted data to an unnecessarily large number of
> attacks whose goal is to recover the plaintext.

Author: ```suggestion
This vulnerability exposes encrypted data to a number of
attacks whose goal is to recover the plaintext.
```
